src/raw.rs

```diff
@@ -105,14 +103,21 @@ fn ignore_data(file_name: &str, crate_blacklist: Blacklist) -> bool {
     false
 }
 
+fn read_file_contents(path: &Path) -> Result<String, ::std::io::Error> {
+    let mut file = File::open(&path)?;
+    let mut buf = String::new();
+    file.read_to_string(&mut buf)?;
+    Ok(buf)
+}
 
 fn read_crate_data(path: &Path) -> Option<Analysis> {
     trace!("read_crate_data {:?}", path);
-    // TODO unwraps
     let t = Instant::now();
-    let mut file = File::open(&path).unwrap();
-    let mut buf = String::new();
-    file.read_to_string(&mut buf).unwrap();
+
+    let buf = read_file_contents(path).or_else(|err| {
```

Comment:
Separated into `read_file_contents` to catch a possible error and report it, however if we were to ignore warning about the error, this could be further simplified with `ok()?`s - should I change that?

Reply:
Nah, seems good to acknowledge the error